static/js/pages/instances/InstanceStatusIcon.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react";
+import { LxdInstance } from "types/instance";
+
+interface Props {
+  instance: LxdInstance;
+}
+
+const InstanceStatusIcon: FC<Props> = ({ instance }) => {
+  // todo: which states are used - can error/unknown/init be removed?
+  const getIconClassForStatus = (status: string) => {
+    return {
+      error: "p-icon--oval-red",
```

Comment:
Let's make this gren on running, grey on stopped and yellow in all other cases.

Reply:
Resolved @ [15828c6](https://github.com/canonical/lxd-ui/commit/15828c618a635e47275a8b2e59458573733c7bc0)